### DISCUSSION TITLE
Frigate with Google Coral

### DISCUSSION THREAD

**Discussion by antsH356** [blakeblackshear/frigate#6220] [General]
Hi All

I have HA installed in Hyper V on a windows pc.  I use frigate and it works really well.  I have since purchased a USB - Google Coral but can't get to it because of no USB pass through in Hyper V (that I can see).  Is there any way I can configure this to work with this current set up of should I build a Linux PC or use a different VM (like - Oracle - VirtualBox )
Any help would be much appreciated.

Ants

**Comment by NickM-27** [comment]
VirtualBox also does not support this, and there is no way without USB passthrough

**Reply by antsH356** [reply]
Thanks for that.  So how should I configure my system to achieve this

**Reply by blakeblackshear** [reply]
Windows is not a supported host OS even with a Linux VM if you want to use anything other than the CPU for decoding and detection. If there are people using it successfully with Frigate, we aren't aware of it.

**Reply by antsH356** [reply]
Thanks- So as per my original question should I start from scratch and turn my pc into a Linx box and reconfigure everything.

**Reply by antsH356** [reply]
I also checked virtual box before I requested help and when i checked in google and chat AI got the following:

Yes, VirtualBox supports USB pass through, which allows you to connect a USB device to a virtual machine (VM) running in VirtualBox.

To use USB pass through, you need to install the VirtualBox Extension Pack, which provides additional features for VirtualBox, including support for USB 2.0 and 3.0 devices. Once you have installed the Extension Pack, you can enable USB pass through for a VM by following these steps:

Make sure the VM is powered off.
Select the VM in the VirtualBox Manager.
Click the "Settings" button.
Click the "USB" tab.
Click the "Add" button to add a new USB filter.
Select the USB device you want to pass through to the VM from the list of available devices.
Click the "OK" button to save the USB filter.
After you have enabled USB pass through for a VM, the device should be available to the VM when you start it up. Note that you may need to install drivers or configure the device within the VM before it can be used.

So even thought this says it is supported will it not work with what I am trying to do.  I'm confused?

**Reply by NickM-27** [reply]
I'd recommend Linux but of course you can give VB toolbox a shot and see if it works.

**Reply by antsH356** [reply]
thanks - ill rebuild a box using just linux
